e2etest/e2etest_suite_test.go

```diff
@@ -98,8 +103,20 @@ var _ = ginkgo.BeforeSuite(func() {
 		testNameSpace = ns
 	}
 
-	if dir := os.Getenv("FRR_CONFIG_DIR"); len(dir) != 0 {
-		frrTestConfigDir = dir
+	if net := os.Getenv("FRR_CONTAINER_NETWORK"); len(net) != 0 {
```

Comment:
You can use os.LookupEnv here. Also, would it make sense to have it as a boolean flag instead of a string?